ChangeLog

```diff
@@ -37,6 +37,7 @@
  * LeastSquaresExpansion, IntegrationExpansion (openturns.experimental)
  * UniformOverMesh (openturns.experimental)
  * GeneralizedExtremeValueValidation (openturns.experimental)
+ * LinthurstDataset
```

Comment:
maybe specify its an use-case module (I also forgot to add Coles, would you mind adding it here as well ?)

Reply:
Ok. Fixed at 0df9c238e